frontend/src/components/enroll/index.js

```diff
@@ -20,64 +23,35 @@ class Enroll extends Component {
       });
   }
 
+  loaded = () => {
+    this.state.loadCounter += 1;
+    if (this.state.loadCounter % 2 === 0)
+      document.getElementsByClassName("resp-iframe")[0].style.height = "400px";
+    else
+      document.getElementsByClassName("resp-iframe")[0].style.height = "1275px";
+  };
+
   render() {
     return (
       <div>
         <SingleCarousel header="Enroll" />
         <div className="container group">
-          <h1 className="mt-5">Enroll</h1>
           <div className="container col-sm-6 float-right mt-5">
             <Markdown id="fontcss" className="mt-3 " source={this.state.text} />
           </div>
-          <div className="container col-sm-6 float-left mt-5">
-            <form action="/" method="post">
-              <div>
-                <label for="name" className="col-xs-4">
-                  Name (required)
-                </label>
-                <input
-                  id="name"
-                  required=""
-                  type="text"
-                  className="col-md-10"
-                />
-              </div>
-              <br />
-              <div>
-                <label for="email" className="col-xs-4">
-                  Email (required)
-                </label>
-                <input
-                  id="email"
-                  required=""
-                  type="email"
-                  className="col-md-10"
-                />
-              </div>
-              <br />
-              <div>
-                <label for="phone" className="col-xs-4">
-                  Phone (required)
-                </label>
-                <input id="phone" type="text" className="col-md-10" />
-              </div>
-              <br />
-
-              <div>
-                <label className="col-xs-4 mt-3 mb-1">Message</label>
-
-                <textarea name="" rows="6" className="col-md-10 mb-4" />
-              </div>
-
-              <div className="col-xs-8 mb-5">
-                <p>
-                  <button className="button" type="submit" value="Submit">
-                    {" "}
-                    <span className="buttonSpan">Submit</span>
-                  </button>
-                </p>
-              </div>
-            </form>
+          <div className="container col-sm-6 mt-5">
+            <iframe
+              src="https://docs.google.com/forms/d/e/1FAIpQLScH_lkw44ikfkHlHpUFVsAtXF6MzElK19xWUyVOP_mJ-ClmHw/viewform?embedded=true"
+              width="100%"
+              height="1250"
+              frameborder="0"
+              marginheight="0"
```

Comment:
`marginHeight` and `marginWidth` here are not doing anything so they can be removed. We can also remove `height` since we are going to set it in the css.